view/frontend/templates/js/replacejs.phtml

```diff
@@ -530,6 +530,7 @@ if (!$block->isOnPageFromWhiteList() && !$block->isMinicartEnabled()) { return;
 
             close: function () {
                 popUpOpen = false;
+                customerData.reload(['boltcart'], true);
```

Comment:
We already invalidated bolt cart in line 542 (543). The purpose of the task is to invalidate magento cart as well.
Looks like we to insert code from jira before line 542 and test it